ethereum/core/src/test/java/org/hyperledger/besu/ethereum/core/encoding/TransactionDecoderTest.java

```diff
@@ -93,4 +94,23 @@ void shouldDecodeWithHighNonce() {
     assertThat(transaction).isNotNull();
     assertThat(transaction.getNonce()).isEqualTo(MAX_NONCE - 1);
   }
+
+  @Test
+  void shouldCalculateCorrectTransactionSize() {
+    for (final String rlp_tx :
```

Comment:
you could use a parameterised test for this

Reply:
see WordsTest for an example - then if the first one fails, the rest will still run because they are effectively separate tests

Reply:
Done!